cache/ttl/cache.go

```diff
@@ -110,11 +134,25 @@ func (cache *Cache) Delete(key string) bool {
 	return true
 }
 
+// Range calls f on every key
```

Comment:
Could add more details here? Like the cache will be deleted if failed in the call